libs/cms/src/lib/models/oneColumnText.model.ts

```diff
@@ -35,9 +36,9 @@ export const mapOneColumnText = ({
   typename: 'OneColumnText',
   id: sys.id,
   title: fields.title ?? '',
-  link: fields.link ? mapLink(fields.link) : null,
+  link: mapReferenceLink(fields.link),
```

Comment:
The term reference link is already used elsewhere, could we maybe find a different name or somehow make the check here whether it is a link and either call mapLink or the generateOrganizationSubpageLinkFromLinkedPage?